liquigraph-core/src/main/java/org/liquigraph/core/io/xml/ChangelogParser.java

```diff
@@ -83,7 +83,7 @@ private void fixUpChangesets(Changelog changelog) {
     }
 
     private String formatErrorMessage(Collection<String> errors) {
-        String separator = "\n\t";
+        String separator =  System.getProperty("line.separator") + "\t";
```

Comment:
We're on Java 7, would you mind using http://docs.oracle.com/javase/8/docs/api/java/lang/System.html#lineSeparator-- instead?